hazelcast/src/main/resources/hazelcast-client-full-example.yaml

```
@@ -162,20 +162,18 @@ hazelcast-client:
   #         kerberos-host: kerb-host-name
   #         kerberos-config-file: kerb.conf
   # * "ssl":
-  #     Lets you configure SSL using the SSL context factory. This feature is available only in Hazelcast
-  #     Enterprise. To be able to use it, encryption should NOT be enabled and you should first implement
-  #     your SSLContextFactory class. Its configuration contains the factory class and SSL properties.
-  #     By default, it is disabled. The following is an example:
+  #     Allows configuring TLS/SSL communication encryption. This feature is available only in Hazelcast
+  #      Enterprise.
+  #      The default configuration usually uses JSSE engine available in Java.
+  #      Consult the Hazelcast documentation on how to integrate with OpenSSL/BoringSSL engines.
```

Minor: These 3 lines are indented with an extra space, maybe we can remove it
```suggestion
  #     Enterprise.
  #     The default configuration usually uses JSSE engine available in Java.
  #     Consult the Hazelcast documentation on how to integrate with OpenSSL/BoringSSL engines.
```